.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+jdk:
+  - oraclejdk11
```

Comment:
Nie lepiej używać OpenJDK?

Reply:
Póki tego nie stawiamy gdzieś to chyba ok, ale żeby nie zapomnieć to później zmienić to już zmieniam